test/testcafe/spec/EnrollAuthenticatorSecurityQuestion_spec.js

```diff
@@ -58,8 +58,8 @@ test.requestHooks(answerRequestLogger, authenticatorEnrollSecurityQuestionMock)(
   await t.expect(radioOptionLabel).eql('Choose a security question');
   // assert Custom Question textbox doesn't show up
   await t.expect(enrollSecurityQuestionPage.isCreateMyOwnSecurityQuestionTextBoxVisible()).notOk();
-  // no signout link at enroll page
-  await t.expect(await enrollSecurityQuestionPage.signoutLinkExists()).notOk();
+  // signout link at enroll page
+  await t.expect(await enrollSecurityQuestionPage.signoutLinkExists()).ok();
```

Comment:
Another occurrence, not sure if this was intended..

@vincentvoong-okta - So this is the screen in Enroll SQ flow where we prompt the user to enter the secret question's answer. With this change, we will have a Back to sign in link in the footer, is this ok?